bindings/protocols/bacnet/index.html

```diff
@@ -1439,7 +1465,7 @@ <h2 id="examples">Examples</h2>
                             },
                             {
                                 "op": [ "writeproperty" ],
-                                "href": "bacnet://5/2,1?commandPriority={writePriority}",
+                                "href": "bacnet://5/2,1?commandPriority={writePriority}{&relinquish}",
```

Comment:
`&` should be outside of the `{}` no?

Reply:
I think it is even more different. I am not an [rfc6570](https://www.rfc-editor.org/rfc/rfc6570) expert  but playing around with https://www.utilities-online.info/uritemplate seems to work fine with

[`bacnet://5/2,1?commandPriority={writePriority}&relinquish={relinquishValue}`](bacnet://5/2,1?commandPriority={writePriority}&relinquish={relinquishValue})

and as input 

```
{
  "writePriority": 16,
  "relinquishValue": true
}
```

creates

`bacnet://5/2,1?commandPriority=16&relinquish=true`